.editorconfig

```diff
@@ -135,6 +135,27 @@ csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_throw_expression = true:suggestion
```

Comment:
@iSazonov, your last commit had 18 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).#requires nested scan tokenizer tests.Correctly resets the first and last tokens in the tokenizer after nested scan in script("#requires -Version 5.0
10","10","10")
```powershell
Expected exactly '10', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 1340
1340:             $tokens[0] | Should -BeExactly $firstToken
```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).#requires nested scan tokenizer tests.Correctly resets the first and last tokens in the tokenizer after nested scan in script("Write-Host 'Hello'
#requires -Version 5.0
7","Write-Host","7
```powershell
Expected exactly 'Write-Host', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 1340
1340:             $tokens[0] | Should -BeExactly $firstToken
```
ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs).#requires nested scan tokenizer tests.Correctly resets the first and last tokens in the tokenizer after nested scan in script("Write-Host 'Hello'
#requires -Version 5.0","Write-Host","Hel
```powershell
Expected exactly 'Write-Host', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1: line 1340
1340:             $tokens[0] | Should -BeExactly $firstToken
```
Requires tests.Parser error.throws ParserException - appId with newline
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParseException' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Requires.Tests.ps1: line 18
18:             { [scriptblock]::Create($command) } | Should -Throw -ErrorId "ParseException"
```
Requires tests.Parser error.throws ParserException - version as character
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParseException' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Requires.Tests.ps1: line 18
18:             { [scriptblock]::Create($command) } | Should -Throw -ErrorId "ParseException"
```